test/backgroundWorker_test.cc

```diff
@@ -2,14 +2,16 @@
 #include <fcntl.h>
 #include <gflags/gflags.h>
 #include <rocksdb/file_system.h>
-#include <rocksdb/plugin/zenfs/fs/fs_zenfs.h>
-#include <rocksdb/plugin/zenfs/fs/zbd_zenfs.h>
+#include <third-party/zenfs/fs/fs_zenfs.h>
```

Comment:
You can use `fs/fs_zenfs.h` here, I added `third-party/zenfs` to CMake build path